data/json/terrain.json

```diff
@@ -66,6 +68,7 @@
         "type" : "terrain",
         "id" : "t_scrap_wall",
         "name": "simple metal wall",
+        "description": "A relatively simple wall made of scrapped together metal, still capable of supporting a roof with enough walls and blocking intruders.",
```

Comment:
Should be `scraped together`

Also recommend moving the comma to a period, e.g

> A relatively simple wall made of scrapped together metal. It's still capable of supporting a roof with enough walls and blocking intruders.

The two ending clauses read funny, I might suggest dropping "enough walls":

> still capable of supporting a roof and keeping intruders out